### PR TITLE
Project rework

### PR DESCRIPTION
**Pull request by hyunhjung**

### CODE REVIEW

File: api/account/account.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+const options = {
+  host: 'localhost',
+  port: 3306,
+  user: 'tokrit',
+  password: '111111',
+  database: 'tokrit'
+}
+const sessionStore = new MySQLStore(options);
+const mysql = require('mysql');
+const tokritDb = mysql.createConnection({
```
Comment (Ethenalee) [inline api/account/account.js]: Any reason, write credential info directly ? For better usage, I think it is better to remove database credentials and import as .env

Comment (hyunhjung) [reply api/account/account.js]: Agreed.

File: api/account/account.js [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+      });
+    });
+  });
+});
+//#endregion
+
+function validateEmail(email) {
+  var ret = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  return ret.test(String(email).toLowerCase());
+}
```
Comment (Ethenalee) [inline api/account/account.js]: Maybe better to keep consistency to ES6 syntax

Comment (hyunhjung) [reply api/account/account.js]: Found this npm package. Might be better to remove this function and use this as we are going to need some other validator functions.
https://www.npmjs.com/package/express-validator

Comment (Ethenalee) [reply api/account/account.js]: yea it might be helpful for not only email validation but also password / username

File: api/account/account.js [hunk 3]
```diff
@@ -0,0 +1,194 @@
+'use strict'
+
+const session = require('express-session');
+const MySQLStore = require('express-mysql-session')(session);
+const bkfd2Password = require('pbkdf2-password');
+const hasher = bkfd2Password();
+const express = require('express');
+const router = express.Router();
+
```
Comment (Ethenalee) [inline api/account/account.js]: want to discuss more/ quite not sure about the structure

**Pull request merged**